app/plugins/views.plugin.js

```diff
@@ -10,9 +10,10 @@
  * @module ViewsPlugin
  */
 
-const path = require('path')
 const Nunjucks = require('nunjucks')
 const Vision = require('@hapi/vision')
+const path = require('path')
```

Comment:
`path` being where it was is intentional, as per [Top of .js Files](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#top-of-js-files)

If you were following that convention there would be a line break after `Vision` then your config 😁